app/src/main/res/values/strings.xml

```diff
@@ -105,9 +105,9 @@
   <string name="receive_withdraw_button">Use LNURL link</string>
 
   <string name="receive_swap_in_button">Show a bitcoin address</string>
-  <string name="receive_swap_in_error_message">There was an error when swapping to an on-chain transaction…\n\nPlease try again.</string>
-  <string name="receive_swap_in_progress_message">Requesting a swap-in bitcoin address…</string>
-  <string name="receive_swap_in_disclaimer_title">Receive on-chain</string>
+  <string name="receive_swap_in_error_message">There was an error while sending this transaction using the Bitcoin network…\n\nPlease try again.</string>
```

Comment:
This "swap-in" error happens when the wallet failed to request a swap to the server, not because a Bitcoin transaction failed, so this translation is not correct.

Reply:
```suggestion
  <string name="receive_swap_in_error_message">There was an error when swapping to a transaction on bitcoin network…\n\nPlease try again.</string> 
```